modules/private-clusters-setting-dns-private.adoc

```diff
@@ -4,13 +4,22 @@
 
 :_mod-docs-content-type: PROCEDURE
 [id="private-clusters-setting-dns-private_{context}"]
-= Setting DNS to private
+= Configuring DNS records to be published in a private zone
 
-After you deploy a cluster, you can modify its DNS to use only a private zone.
+For all {product-title} clusters, whether public or private, DNS records are published in a public zone by default.
+
+You might use a public zone for DNS records in cases where you want to have a private cluster while still allowing clients to look up names for applications running on that cluster. For example, if an organization has machines that connect to the public internet, and establish VPN connections for certain private IP ranges in order to connect to private services. The DNS lookups from these machines use the public DNS to determine the private addresses of those services, and then connect to the private addresses over the VPN.
```

Comment:
```suggestion
Even in a private cluster, you might keep the public zone for DNS records because it allows clients to resolve DNS names for applications running on that cluster. For example, an organization has machines that connect to the public internet, and establish VPN connections to connect to private services. The DNS lookups from these machines use the public DNS to determine the private addresses of those services, and then connect to the private addresses over the VPN.
```

Reply:
Yeah, the current "for example" sentence fragment ends rather jarringly; modifying it to be a full sentence helps with that.  I'd also suggest removing the comma, and adding a "can" to express that this is a general possibility:

> For example, an organization can have machines that connect to the public internet and then establish VPN connections for certain private IP ranges in order to connect to private IP addresses.  

The text might also be clearer if you replaced "services" with another word, such as "applications" or "IP addresses", because we're not talking about Kubernetes services (which generally are unreachable outside the cluster even if you know their IP addresses); rather, we're talking about applications exposed using OpenShift routes and a private load-balancer or router virtual-IP address.